pkg/users/account-create-dialog.js

```diff
@@ -88,6 +105,33 @@
                     </FlexItem>
                 </Flex>
             </FormGroup>
+            <FormGroup label={_("Home directory")}
+                       helperTextInvalid={errors && errors.home_dir}
+                       validated={(errors && errors.home_dir) ? "error" : "default"}
+                       fieldId="accounts-create-user-home-dir">
+                <TextInput id="accounts-create-user-home-dir"
+                    onChange={value => change("home_dir", value)}
+                    placeholder={_("Path to directory")}
+                    value={state.home_dir} />
+            </FormGroup>
+            <FormGroup label={_("Shell")}
+                       helperTextInvalid={errors && errors.shell}
+                       validated={(errors && errors.shell) ? "error" : "default"}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18234-20230131-135835-5b43f8e5-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)